lib/web/urbandictionary.py

```diff
@@ -17,45 +17,45 @@ def __init__(self, word, url, description, example, author):
         self.example = example
         self.author = author
         self._parse()
-    
+
     def _parse(self):
-        
+
         """ Strips links from the definition and gathers them in a links property.
         """
-        
+
         p1 = "\[.*?\](.*?)\[\/.*?\]"
         p2 = "\[(.*?)\]"
         self.links = []
         for p in (p1,p2):
             for link in re.findall(p, self.description):
                 self.links.append(link)
             self.description = re.sub(p, "\\1", self.description)
-            
+
         self.description = self.description.strip()
-    
+
     def __str__(self):
-        
+
         return self.description
 
 class UrbanDictionaryError(Exception):
     pass
 
 class UrbanDictionary(list):
-    
+
     def __init__(self, q, cached=True):
-        
+
         url = "http://api.urbandictionary.com/soap?wsdl"
         key = "91cf66fb7f14bbf7fb59c7cf5e22155f"
```

Comment:
OT - but, having keys in the code doesn't seem like the best idea - but API key management is almost a whole separate project in itself (something our IDE should maybe help with).